app/scripts/head/start.js

```diff
@@ -1,10 +1,10 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+'use strict';
 
 (function () {
-  'use strict';
-
+  
```

Comment:
the extra line break is due to use strict.
for example here: ![](http://v14d.com/i/5574d4f24e37d.jpg)

if we remove use strict we don't need the extra line break anymore. Also see my comment regarding the JSCS rule.